storagenode/console/service.go

```diff
@@ -97,117 +102,139 @@ func NewService(log *zap.Logger, consoleDB DB, bandwidth bandwidth.DB, pieceStor
+	data.DiskSpace = DiskSpaceInfo{
+		Used:      memory.Size(spaceUsege).GB(),
+		Available: s.allocatedDiskSpace.GB(),
+	}
+
+	data.Bandwidth = BandwidthInfo{
+		Egress: Egress{
+			Repair: bandwidthUsage.GetRepair,
+			Audit:  bandwidthUsage.GetAudit,
+			Usage:  bandwidthUsage.Get,
+		},
+		Ingress: Ingress{
+			Repair: bandwidthUsage.PutRepair,
+			Usage:  bandwidthUsage.Put,
+		},
+		Used:      memory.Size(bandwidthUsage.Total()).GB(),
+		Available: s.allocatedBandwidth.GB(),
+	}
+
+	return data, nil
+}
+
+// Satellite encapsulates satellite related data
+type Satellite struct {
+	ID             storj.NodeID         `json:"satelliteID"`
+	StorageDaily   []storageusage.Stamp `json:"storageDaily"`
+	BandwidthDaily []BandwidthUsed      `json:"bandwidthDaily"`
+	Audit          reputation.Metric    `json:"audit"`
+	Uptime         reputation.Metric    `json:"uptime"`
 }
 
-// GetUsedStorageTotal returns all info about storagenode disk space usage
-func (s *Service) GetUsedStorageTotal(ctx context.Context) (_ *DiskSpaceInfo, err error) {
+// GetSatelliteData returns satellite related data
+func (s *Service) GetSatelliteData(ctx context.Context, satelliteID storj.NodeID) (_ *Satellite, err error) {
 	defer mon.Task()(&ctx)(&err)
+	// get current month edges
+	from, to := date.MonthBoundary(time.Now())
 
-	spaceUsed, err := s.pieceStore.SpaceUsedForPieces(ctx)
+	bandwidthDaily, err := s.consoleDB.Bandwidth().GetDaily(ctx, satelliteID, from, to)
 	if err != nil {
-		return nil, err
+		return nil, SNOServiceErr.Wrap(err)
 	}
 
-	return &DiskSpaceInfo{Available: s.allocatedDiskSpace.Int64() - spaceUsed, Used: spaceUsed}, nil
-}
-
-// GetUsedStorageBySatellite returns all info about storagenode disk space usage
-func (s *Service) GetUsedStorageBySatellite(ctx context.Context, satelliteID storj.NodeID) (_ *DiskSpaceInfo, err error) {
-	defer mon.Task()(&ctx)(&err)
+	storageDaily, err := s.storageUsageDB.GetDaily(ctx, satelliteID, from, to)
+	if err != nil {
+		return nil, SNOServiceErr.Wrap(err)
+	}
 
-	spaceUsed, err := s.pieceStore.SpaceUsedBySatellite(ctx, satelliteID)
+	rep, err := s.reputationDB.Get(ctx, satelliteID)
 	if err != nil {
-		return nil, err
+		return nil, SNOServiceErr.Wrap(err)
 	}
 
-	return &DiskSpaceInfo{Available: s.allocatedDiskSpace.Int64() - spaceUsed, Used: spaceUsed}, nil
+	return &Satellite{
+		ID:             satelliteID,
+		StorageDaily:   storageDaily,
+		BandwidthDaily: bandwidthDaily,
+		Audit:          rep.Audit,
+		Uptime:         rep.Uptime,
+	}, nil
 }
 
-// GetWalletAddress return wallet address of node operator
-func (s *Service) GetWalletAddress(ctx context.Context) string {
+// GetAllSatellitesData returns bandwidth and storage daily usage consolidate
+// among all satellites from node's trust pool
+func (s *Service) GetAllSatellitesData(ctx context.Context) (_ *Satellite, err error) {
 	defer mon.Task()(&ctx)(nil)
-	return s.walletAddress
-}
+	// get current month edges
```

Comment:
I don't think that this comment is needed as the `date.MonthBoundary` function documentation already mentions what does this.